css/styles.css

```diff
@@ -135,28 +135,21 @@ a.icon-link {
   margin-bottom: 1rem;
 }
 
-.text-field .mdc-notched-outline .mdc-notched-outline__trailing {
-  border-radius: 0 28px 28px 0;
-}
 
 .text-field .mdc-notched-outline .mdc-notched-outline__leading {
-  border-radius: 28px 0 0 28px;
   width: 28px;
 }
 
 .text-field.text-field--textarea .material-icons {
   top: 16px;
 }
 
-/* .text-field--textarea.text-field .mdc-notched-outline .mdc-notched-outline__trailing {
-  border-radius: 0 56px 56px 0;
+/* the border around the input fields */
+.text-field .mdc-notched-outline .mdc-notched-outline__leading,
+.text-field .mdc-notched-outline .mdc-notched-outline__trailing {
+  border-radius: 0;
 }
```

Comment:
Apologies, I wasn't clear enough in the task description.

What I meant is that the border-radius overrides should be removed, so the default mdc border radius remains, not to set them all to zero.

So, essentially, remove these lines entirely.